catalog/kyverno/releases.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kyverno
+  namespace: kyverno
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: kyverno
+      version: v2.6.5
+      sourceRef:
+        kind: HelmRepository
+        name: kyverno
+      interval: 1m
+  install:
+    crds: Create
```

Comment:
I would always use CreateReplace, so existing crds are upgraded

Reply:
it's now changed to `CreateReplace`.